hmda-dashboard/src/main/resources/application.conf

```diff
@@ -0,0 +1,63 @@
+include "cors.conf"
+
+akka {
+  loglevel = info
+  log-dead-letters = off
+  log-dead-letters-during-shutdown = off
+}
+
+akka.http.server.request-timeout = infinite
+
+server {
+  bindings {
+    address: "0.0.0.0"
+    address: ${?SERVER_BIND_ADDRESS}
+
+    port: 9999
+    port: ${?BIND_PORT}
+  }
+
+  ask-timeout: 10
+
+}
+
+// see keys dbconfig and db in common/src/main/resources/application.conf
+// we provide project overrides for development
+dbconfig {
```

Comment:
lets take this entire `dbconfig` block out. This will be read from `common/application.conf`.